app/javascript/article-form/components/__tests__/LoadingPreview.test.jsx

```diff
@@ -0,0 +1,19 @@
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
+import { axe } from 'jest-axe';
+import { LoadingPreview } from '..';
+
+describe('<LoadingPreview />', () => {
+  it('should have no a11y violations', async () => {
+    const { container } = render(<LoadingPreview />);
+    const results = await axe(container);
+
+    expect(results).toHaveNoViolations();
+  });
+
+  it('should render', () => {
```

Comment:
(nitpick / not blocking) - maybe we should check it renders with the "cover" variant too?

Reply:
Added test for `should render with cover image`